cypress/utils/data_utils.ts

```diff
@@ -160,6 +161,18 @@ export function getRandomCredentialsData(
         };
     }
 
+    if (type === CredentialType.jiraToken) {
+        if (useTestingAccount) {
+            key = Cypress.env("jira_key");
```

Comment:
This new key should be also added in the `cypress.config.ts` file

Reply:
Or it can be provided as a system variable, which are actually used by Jenkins job in most cases

Reply:
Right, I'll add empty key